{"name": "test_valid_auth[performance_glitch_user-secret_sauce]", "status": "failed", "statusDetails": {"message": "AssertionError: Кнопка Logout не найдена\nassert None == 'Logout'\n +  where None = <bound method MainPage.logout_button_text of <pages.main_page.MainPage object at 0x000002664A4C4B80>>()\n +    where <bound method MainPage.logout_button_text of <pages.main_page.MainPage object at 0x000002664A4C4B80>> = <pages.main_page.MainPage object at 0x000002664A4C4B80>.logout_button_text\n +      where <pages.main_page.MainPage object at 0x000002664A4C4B80> = <pages.application.Application object at 0x000002664A4FA4C0>.main_page\n +  and   'Logout' = ButtonTexts.LOGOUT_BUTTON_TEXT", "trace": "self = <tests.test_auth.TestAuth object at 0x000002664A4C44F0>, email = 'performance_glitch_user', password = 'secret_sauce'\napp = <pages.application.Application object at 0x000002664A4FA4C0>\n\n    @allure.story(\"Авторизация\")\n    @allure.severity(\"blocker\")\n    @pytest.mark.parametrize(\n        \"email, password\",\n        (\n            (Users.STANDARD_USERNAME, Users.PASSWORD),\n            (Users.PROBLEM_USERNAME, Users.PASSWORD),\n            (Users.PER_GLITCH_USERNAME, Users.PASSWORD),\n        ),\n    )\n    def test_valid_auth(self, email, password, app):\n        \"\"\"\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        \"\"\"\n        app.open_main_page()\n        app.login.auth(email, password)\n>       assert (\n            app.main_page.logout_button_text() == ButtonTexts.LOGOUT_BUTTON_TEXT\n        ), \"Кнопка Logout не найдена\"\nE       AssertionError: Кнопка Logout не найдена\nE       assert None == 'Logout'\nE        +  where None = <bound method MainPage.logout_button_text of <pages.main_page.MainPage object at 0x000002664A4C4B80>>()\nE        +    where <bound method MainPage.logout_button_text of <pages.main_page.MainPage object at 0x000002664A4C4B80>> = <pages.main_page.MainPage object at 0x000002664A4C4B80>.logout_button_text\nE        +      where <pages.main_page.MainPage object at 0x000002664A4C4B80> = <pages.application.Application object at 0x000002664A4FA4C0>.main_page\nE        +  and   'Logout' = ButtonTexts.LOGOUT_BUTTON_TEXT\n\ntests\\test_auth.py:31: AssertionError"}, "description": "\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        ", "attachments": [{"name": "screenshot", "source": "26095278-77b4-4b76-9d4e-0f2293e05792-attachment.png", "type": "image/png"}, {"name": "log", "source": "bbd6e7b1-655c-431b-b71b-74203217fc11-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "224640b7-e0c9-42f6-b5f6-a23f549faba4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "email", "value": "'performance_glitch_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1615299378239, "stop": 1615299384705, "uuid": "6d3d1818-1d2d-4662-a4c1-d91ea7b86b45", "historyId": "8df94a67bfcca65d5cf57a3d35530f9b", "testCaseId": "52b3ff36bc4dce34c6b761d4e4c54109", "fullName": "tests.test_auth.TestAuth#test_valid_auth", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "LAPTOP-Q6599UFH"}, {"name": "thread", "value": "19868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}
